_includes/1-topic-landing.html

```diff
@@ -9,13 +9,19 @@
     {% include action-bar.html action_text="Apply for Education Benefits" action_url="/education/apply-for-education-benefits/" action_class="va-action-bar--header" %}
   {% endif %}
 
+  {% if page.collection == 'health-care' and page.showactionbar == true %}
+    {% include action-bar.html action_text="Apply for VA Health Care" action_url="/health-care/apply/healthcare/apply/application" action_class="va-action-bar--header" %}
```

Comment:
Not sure what this is for, but the url doesn't look right to me. It should just be `/healthcare/apply/application`. I know that right now we have pages under `health-care`, but I think we decided that moving forward we're changing that to `healthcare`, no hyphen. Wondering if that means we should also rename all references to pages called `health-care`.